api/src/main/java/io/opentelemetry/context/NoopScope.java

```diff
@@ -19,11 +19,16 @@
 /** A {@link Scope} that does nothing when it is created or closed. */
 public final class NoopScope implements Scope {
```

Comment:
Where is this used? Can we remove it? Probably in a separate PR.

Reply:
It's used in `DefaultTagger`: https://github.com/open-telemetry/opentelemetry-java/blob/bfe56101d126ce82fca213eecf38008412e2bd01/api/src/main/java/io/opentelemetry/tags/DefaultTagger.java#L110. I think it needs to be public, otherwise we need to duplicate the code on noop scope implementation in `DefaultTagger`.